libafl/src/bolts/shmem.rs

```diff
@@ -979,3 +979,61 @@ pub mod win32_shmem {
         }
     }
 }
+
+/// A cursor around [`ShMem`] that immitates [`std::io::Cursor`]. Notably, this implements [`Write`] for [`ShMem`].
+#[cfg(feature = "std")]
```

Comment:
Looking at `core::fmt::Write`, I'm not sure it would be a good fit. It seems like it's just for formatting strings?

I don't know why the `Write` trait would required std in the first place, but there is not much I can do here I think...

Reply:
Ah yes you're right it's for unicode stuff, you're using it for bytes, I guess?
Still, the Cursor itself wouldn't need to be `std`

Reply:
I removed the std requirement for the struct itself btw :)